architecture/core_concepts/routes.adoc

```diff
@@ -302,6 +303,27 @@ ports that the router is listening on, `ROUTER_SERVICE_SNI_PORT` and
 they are unique on the machine. These ports will not be exposed externally.
 ====
 
+[[strict-sni]]
+== HAProxy Strict SNI
+
+By default, when a host does not resolve to a route in a HTTPS or TLS
+SNI request, the default certificate is returned to the caller as part of the
+503 response. This exposes the default certificate and can pose security
+concerns. HAProxy `strict-sni` option to bind suppresses use of the
+default certificate.
+
+The `ROUTER_STRICT_SNI` environment variable controls bind processing. When set to `true` or
+`TRUE`, `strict-sni` is added to the HAProxy bind. Default is `false`.
+
+The option can be set when the router is created or added later.
+
+----
+$ oc adm router --strict-sni
```

Comment:
Just saw your once comment. I did not realize it ran as root, but I see the # oc adm router --strict-sni in the other file. Sorry, I misspoke!